src/common/log/loggers.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+/**
+ * Formatter for the Output channel.
+ */
+import {LogHelper, LogLevel} from "./logHelper";
+import {OutputChannel} from "vscode";
+
+export interface ILogger {
+    logMessage: (message: string, formatMessage?: boolean) => void;
+    logError: (errorMessage: string, error?: any, logStack?: boolean) => void;
+    logInternalMessage?: (logLevel: LogLevel, message: string) => void;
+}
+
+export class OutputChannelLogger implements ILogger {
```

Comment:
Would it be a good idea to move this class to the extension folder since it references "vscode"?